### PR TITLE
[DE] Add `Brand_identity_guidelines` and `Client/Interface/Cookie`; update `Gameplay`, `People` and `Help_centre`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
Notice for reviewers: I've also applied some styling changes in `Help_centre`, because I think that it improves readability.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/de.md [hunk 1]
```diff
@@ -1,13 +1,10 @@
----
-outdated: true
-outdated_since: 7880a47c4e5df0bb9470cdd8edc48daa42e6f277
----
-
 # Menschen
```
Comment (respektive) [inline wiki/People/de.md:1]: I'm thinking maybe this should be translated as `Personen` or `Leute`? I think both work better than `Menschen`. :thinking:

Comment (The-Last-Cookie) [reply wiki/People/de.md:1]: Yeah, good point. I also thought about this myself. The main page uses `Personen`, so I'm going to stick to this. `Leute` sounds a tiny bit too casual in my opinion.

```suggestion
# Personen
```

File: wiki/People/de.md [hunk 2]
```diff
@@ -1,13 +1,10 @@
----
-outdated: true
-outdated_since: 7880a47c4e5df0bb9470cdd8edc48daa42e6f277
----
-
-# Menschen
+# Personen
 
 Indexseite für Artikel über Menschen oder Gruppen.
```
Comment (respektive) [inline wiki/People/de.md:3]:
```suggestion
Indexseite für Artikel über Personen oder Gruppen.
```
just noticed this should be changed as well then whoops

Comment (The-Last-Cookie) [inline wiki/People/de.md:3]: Actually this should be changed too, shouldn't it?

**Review by respektive — DISMISSED** [review]

**Comment by The-Last-Cookie** [comment]
TODO: I need to add in the (unexpected) changes from #7194 . In particular, this line here, above `website` in the table:
```markdown
| [Tournament bans](/wiki/Help_centre/Tournament_bans) | All about tournament bans: overview, common reasons, and cooldowns |
```

**Review by respektive — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**